dev/tests/integration/testsuite/Magento/Contact/Controller/IndexTest.php

```diff
@@ -24,7 +24,7 @@ public function testPostAction()
         $this->assertRedirect($this->stringContains('contact/index'));
         $this->assertSessionMessages(
             $this->contains(
-                "Thanks for contacting us with your comments and questions. We'll respond to you very soon."
+                "Thanks for contacting us with your comments and questions. We&#039;ll respond to you very soon."
```

Comment:
Not sure that it will be good idea to make it escaped there. Have you checked this change?

Reply:
Yes, I have checked this manually. On Contact Form It works as expected:
<img width="616" alt="contact_form_html_entities" src="https://user-images.githubusercontent.com/5747062/32606640-b50fa236-c55e-11e7-8579-e3f824dd3cbd.png">

Since this test is testing the controller I assume that it is valid to check for the exact html we expect.